plugins/graphql-kotlin-gradle-plugin/src/test/kotlin/com/expediagroup/graphql/plugin/gradle/tasks/GraphQLGenerateClientTaskIT.kt

```diff
@@ -37,6 +37,9 @@ class GraphQLGenerateClientTaskIT : WireMockAbstractIT() {
         val sourceDirectory = File("src/integration/download-sdl-timeout")
         sourceDirectory.copyRecursively(testProjectDirectory)
 
+        // version catalog setup
+        File("../../gradle/libs.versions.toml").copyTo(File(testProjectDirectory, "gradle/libs.versions.toml"))
```

Comment:
Alternative would be to create `settings.gradle.kts` files in each integration test directory that would load version catalog from path that is specified as env variable. I think this approach is simpler